package.json

```diff
@@ -59,6 +62,7 @@
     "csv-parse": "3.0.0",
     "csv-parser": "1.12.1",
     "eslint": "5.7.0",
+    "eslint-plugin-compat": "2.7.0",
```

Comment:
Not sure we need this. This plugin warns when you use potentially unsupported browser APIs which won't help here.